src/app/components/evaluation-info/evaluation-info.component.html

```diff
@@ -22,7 +22,12 @@ <h1 class="nowrap" [ngStyle]='getMoveStyle()'>{{ displayedMoveNumber }}</h1>
     <button aria-label="Increment Displayed Move" mat-icon-button [disabled]="isIncrementDisabled()" (click)="incrementDisplayedMove()" title="Increment Move">
       <mat-icon class="icon-color" svgIcon="ic_chevron_right"></mat-icon>
     </button>
-    <h1 class="space-right">of&nbsp;&nbsp;&nbsp;{{ getMaxMoveNumber() }}</h1>
+    <h1>&nbsp;of&nbsp;&nbsp;&nbsp;{{ getMaxMoveNumber() }}</h1>
+    <div class="move-div" *ngIf="selectedEvaluationId && displayedMoveNumber === currentMoveNumber">

```

Comment:
should be
*ngIf="selectedEvaluationId && canIncrementMoveNumber && displayedMoveNumber === currentMoveNumber"